src/features/Room/RoomEntryModal/RoomEntryModal.module.scss

```
@@ -41,17 +42,17 @@
 
   &__title {
     @include m.text-style('lg', 700);
-    color: f.color(gray-900);
+    color: var(--text);
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)